docs/source/guides/configs/k8s.sh

```diff
@@ -0,0 +1,300 @@
+#!/bin/sh
+
+# the purpose of this scripts:
+# - setup minikube, kubectl, helm if doesn't exists through binary
+# - Run through most of configurations setup for k8s cluster.
+#
+# NOTES:
+# - This scripts should run as-is, meaning this script will run within BentoML documents scope.
+# - Example BentoService: aarnphm/bentoml-sentiment-analysis:latest
+# - Assume that users before running this script to setup VirtualBox
+# - This only works on POSIX system as we will install binary for most of dependency (skip if users already have dependencies)
+
+# Contains functions from https://github.com/client9/shlib
+set -e
+
+LOG_LEVEL=2
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+K8S_CONFIG="${GIT_ROOT}/docs/source/guides/configs"
+BIN_DIR=/usr/local/bin
+
+TMP_DIR=$(mktemp -d)
+
+trap 'rm -rf ${TMP_DIR}' EXIT
+
+usage () {
+  this="$1"
+  cat <<EOF
+${this}: Setup BentoService on K8s with Prometheus-Grafana stack
+
+Usage: ${this} [-d | -t] [-h]
+  -t enables trace.
+  -d enables debug logging.
+  -h show this help message
+EOF
+	exit 2
+}
+
+main() {
+  parse_args "$@"
+  _NAMESPACE=bentoml
+  if ! is_command virtualbox; then
+    log_err "In order to run the script, you need to install virtualbox. Exiting."
+    return 1
+  fi
+  log_warn "While this scripts help ease the some of the pain for the demo, it is recommend for users to go through setup steps provided in the guide."
+
+  log_info "Setting up binary..."
+  install_binary
+  cd "${K8S_CONFIG}"
+
+  log_info "Setting up minikube..."
+  if [ "$(minikube config get driver)" != "virtualbox" ]; then
+      minikube config set driver virtualbox
+  fi
+  minikube delete && minikube start \
```

Comment:
this may delete users' existing minikube vm, right?

Reply:
This will delete users' existing minikube vm, since we need to setup minikube accordingly to use the helm chart. I will add warning to users when using this scripts.